### PR TITLE
READY: Add display endpoint

### PR DESCRIPTION
**Pull request by MaxVanDeursen**
Implements [#80](https://github.com/vandenheuvel/tribler/issues/80) from the Tribler repository.

### CODE REVIEW

File: FakeTriblerAPI/models/multichain_block.py [hunk 1]
```diff
@@ -1,78 +1,35 @@
...
-            self.public_key_responder = my_id
-
-        self.previous_hash_requester = 'a' * 20
-        self.previous_hash_responder = 'b' * 20
-        self.signature_requester = 'c' * 20
-        self.signature_responder = 'd' * 20
-
+    def __init__(self, my_id=None, timestamp=0, last_block=None):
+        self.public_key = my_id
+        self.sequence_number = 1 if last_block is None else last_block.sequence_number + 1
```
Comment (ghabbenjansen) [inline FakeTriblerAPI/models/multichain_block.py:10]: Please change this to:
```python
self.sequence_number = last_block.sequence_number + 1 if last_block else 1
```
This way, an equation with None is not needed

**Review by ghabbenjansen — APPROVED** [review]

File: FakeTriblerAPI/endpoints/display_endpoint.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    Handle HTTP requests for the trust display.
+    """
+
+    isLeaf = True
+
+    @staticmethod
+    def return_error(request, status_code=http.BAD_REQUEST, message="your request seems to be wrong"):
+        """
+        Return a HTTP Code with the given message.
+        :param request: the request which has to be changed
```
Comment (vandenheuvel) [inline FakeTriblerAPI/endpoints/display_endpoint.py]: Please add a newline before the argument list

**Review by ghabbenjansen — CHANGES_REQUESTED** [review]
Please take a look at my suggestion

File: FakeTriblerAPI/endpoints/display_endpoint.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        :return: the error message formatted in JSON
+        """
+        request.setResponseCode(status_code)
+        return json.dumps({"error": message})
+
+    def render_GET(self, request):
+        """
+        Process the GET request which retrieves the information used by the GUI Trust Display window.
+        .. http:get:: /display?focus_node=(string: public key)&neighbor_level=(int: neighbor_level)
+        A GET request to this endpoint returns the data from the multichain. This data is retrieved from the multichain
```
Comment (vandenheuvel) [inline FakeTriblerAPI/endpoints/display_endpoint.py]: Please add some newlines in this comment block.

File: FakeTriblerAPI/tribler_data.py [hunk 4]
```diff
@@ -87,6 +88,13 @@ def generate(self):
             }
         }}
 
+        # Create display information
```
Comment (vandenheuvel) [inline FakeTriblerAPI/tribler_data.py]: I believe this inline comment is clear from the context.

**Review by vandenheuvel — CHANGES_REQUESTED** [review]

**Pull request closed without merging**